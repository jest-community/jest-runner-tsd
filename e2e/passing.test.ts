import {expect, test} from '@jest/globals';
import runJest from './runJest';

test('works with passing test', async () => {
  expect(await runJest('passing')).toMatchSnapshot();
});
