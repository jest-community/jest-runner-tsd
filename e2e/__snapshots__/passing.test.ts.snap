// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works with passing test 1`] = `
"PASS ./index.test.ts
  âœ“ tsd typecheck
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
